test/bintray/test

```diff
@@ -0,0 +1,36 @@
+#!/bin/sh
+set -u
+
+stdout=$(mktemp beaver.test.stdout.XXXXXXXX)
+stderr=$(mktemp beaver.test.stderr.XXXXXXXX)
+expout=$(mktemp beaver.test.expout.XXXXXXXX)
```

Comment:
Can you explain these lines?  I'm not sure I follow the intention.

Reply:
I'm just creating temporary files to be used by the tests...